uint32 quad_id
geometry_msgs/PoseStamped pose
